src/sagemaker/session.py

```diff
@@ -195,17 +195,7 @@ def train(self, image, input_mode, input_config, role, job_name, output_config,
                     a directory in the Docker container.
                 * 'Pipe' - Amazon SageMaker streams data directly from S3 to the container via a Unix-named pipe.
 
-            input_config (str or dict or sagemaker.session.s3_input): Information about the training data.
-                This can be one of three types:
-
-                * (str) - the S3 location where training data is saved.
-                * (dict[str, str] or dict[str, sagemaker.session.s3_input]) - If using multiple channels for
-                    training data, you can specify a dict mapping channel names
-                    to strings or :func:`~sagemaker.session.s3_input` objects.
-                * (sagemaker.session.s3_input) - channel configuration for S3 data sources that can provide
-                    additional information about the training dataset. See :func:`sagemaker.session.s3_input`
-                    for full details.
-
+            input_config (list): An array of Channel objects. Each channel is a named input source.
```

Comment:
Could you add a link to the botocore docs?

Reply:
There are no links to `botocore` docs in this module. How about putting a link to `Channel` [API document](https://docs.aws.amazon.com/sagemaker/latest/dg/API_Channel.html) like [session.py#L744](https://github.com/aws/sagemaker-python-sdk/blob/b45d79c/src/sagemaker/session.py#L744)?

Reply:
I meant the [link](https://botocore.readthedocs.io/en/latest/reference/services/sagemaker.html#SageMaker.Client.create_training_job) you referenced in your PR description - since there's more information there about `InputDataConfig`, I think it'd be nice to make it clear where to find more information. A link to the Channel API doc would be a nice addition as well

Reply:
I see. I added the link to the description in 3671efe.